cfg/atari2600.cfg

```diff
@@ -0,0 +1,22 @@
+# Atari VCS 2600 linker configuration file for cc65
+#
+# Florent Flament (contact@florentflament.com), 2017
+
+SYMBOLS {
+    __STACKSIZE__: type = weak, value = $0010;   # 16 Bytes system stack
+}
+
+MEMORY {
+    RAM: file = "", start = $0080, size = $0080 - __STACKSIZE__, define = yes;
+    ROM: file = %O, start = $F000, size = $1000;
```

Comment:
As I understand it, after a PROM (and flash memory, too) is erased, all of its bits are %1.  PROM programming software takes advantage of that.  They don't write $FF bytes; instead, they just skip the location.  Therefore, it is useful to fill unused space with $FF when creating image files for ROMs.